src/ring_theory/dedekind_domain/adic_valuation.lean

```diff
@@ -333,4 +333,71 @@ instance adic_completion.has_lift_t : has_lift_t K (v.adic_completion K) :=
 /-- The ring of integers of `adic_completion`. -/
 def adic_completion_integers : valuation_subring (v.adic_completion K) := valued.v.valuation_subring
 
+variables (R K)
+
+lemma adic_completion.is_integer (x : v.adic_completion K) :
+  x ∈ v.adic_completion_integers K ↔ (valued.v x : ℤₘ₀) ≤ 1 :=
+iff.rfl
+
+section algebra_instances
+
+@[priority 100] instance adic_valued.has_uniform_continuous_const_smul' :
+  @has_uniform_continuous_const_smul R K v.adic_valued.to_uniform_space
```

Comment:
The reason you need to supply the parameter `v.adic_valued.to_uniform_space` twice is that there is an error in our definition of `has_uniform_continuous_const_smul`.

I have fixed this in #17063. Once that is merged, you should merge master here and then you will no longer have to duplicate this argument.